api/namex/resources/events.py

```diff
@@ -156,8 +156,9 @@ def get(nr):
                 user_action = "Undo Decision"
             if e_dict["action"] == "nro_update" and (e_dict["stateCd"] == State.APPROVED or e_dict["stateCd"] == State.REJECTED or e_dict["stateCd"] == State.CONDITIONAL):
                 user_action = "Updated NRO"
-            if e_dict["action"] == "post" and "comment" in e_dict["jsonData"]:
+            if e_dict["action"] == "post" and ((json_data := json.loads(e_dict["jsonData"])) and "comment" in json_data):
                 user_action = "Staff Comment"
+                nr_event_info['comment'] = json_data["comment"]
```

Comment:
simple! Should be able to do the UI side under the assumption this prop name will stay the same :) 
Thanks!